packages/docregistry/src/default.ts

```diff
@@ -227,12 +227,14 @@ export class TextModelFactory implements DocumentRegistry.CodeModelFactory {
    * Create a new model.
    *
    * @param languagePreference - An optional kernel language preference.
+   * @param modelDB - An optional modelDB.
```

Comment:
We might as well add the optional `isInitialized` param to the docstring too?

Reply:
I'm gonna wait for the tests to finish and I'll add a commit.